providers/fab/src/airflow/providers/fab/www/views.py

```diff
@@ -71,7 +71,7 @@ def index(self):
             response = make_response(redirect(f"{conf.get('api', 'base_url', fallback='/')}", code=302))
 
             secure = bool(conf.get("api", "ssl_cert", fallback=""))
```

Comment:
Could you also change the `secure` flag here to something similar in rst doc update?

Something like this:
```
secure = request.base_url.scheme == "https" or bool(conf.get("api", "ssl_cert", fallback=""))
```